src/mbgl/sprite/sprite_parser.cpp

```diff
@@ -24,7 +25,10 @@ SpriteImagePtr createSpriteImage(const PremultipliedImage& image,
         ratio <= 0 || ratio > 10 ||
         srcX >= image.size.width || srcY >= image.size.height ||
         srcX + width > image.size.width || srcY + height > image.size.height) {
-        Log::Error(Event::Sprite, "Can't create sprite with invalid metrics");
+        Log::Error(Event::Sprite, "Can't create sprite with invalid metrics: %" PRIu32 "x%" PRIu32 "@%" PRIu32 ",%" PRIu32 " in %" PRIu32 "x%" PRIu32 "@%sx sprite",
```

Comment:
Sooo, the GLFW `g++5` builds and `android-ndk/arm-9-r12b/bin/arm-linux-androideabi-clang++` don't like this, although the QT `g++5` builds and other clang++ builds are fine... Is there a better way to handle this?